mkdocs_macros/plugin.py

```diff
@@ -6,23 +6,25 @@
 # MIT License
 # --------------------------------------------
 
+import importlib
 import os
-import traceback
 from copy import copy
-import importlib
-
 
 import yaml
-from jinja2 import (Environment, FileSystemLoader, TemplateSyntaxError,
-                    Undefined, DebugUndefined, StrictUndefined)
-from mkdocs.plugins import BasePlugin
+from jinja2 import (
+    Environment, FileSystemLoader, Undefined, DebugUndefined, StrictUndefined,
+)
 from mkdocs.config import config_options
 from mkdocs.config.config_options import Type as PluginType
+from mkdocs.plugins import BasePlugin
+from mkdocs.structure.pages import Page
 
-
-from .util import (install_package, parse_package, trace, debug,
-        update, SuperDict, import_local_module, format_chatter, LOG)
-from .context import define_env
+from mkdocs_macros.errors import format_error
+from mkdocs_macros.context import define_env
+from mkdocs_macros.util import (
```

Comment:
Is there a specific reason why avoiding relative calls within the package?

Reply:
These changes were automatically applied by my PyCharm IDE. The reasoning to avoid relative imports might perhaps be considered a matter of taste. However, I personally try to keep imports always absolute because:

* That simplifies refactoring,
* Makes things more explicit rather than implicit,
* Makes it easier to find a Python module based on the import path — no need to additionally look into where the importing module is located.